cpp/ql/src/change-notes/2023-12-04-incorrectly-checked-scanf.md

```diff
@@ -0,0 +1,4 @@
+---
+category: new-query
```

Comment:
This should make CI happy 🤞. [Here's](https://github.com/github/codeql/blob/main/docs/change-notes.md) the description of the change-note format.

```suggestion
category: newQuery
```